scripts/test_examples.sh

```diff
@@ -1,51 +1,69 @@
 #!/usr/bin/env bash
 
 TMP_FILE="/tmp/out"
+EXITCODE_FILE="/tmp/exit"
 exitCode=0
 PREV_IFS="$IFS"
+RUNTIME_LIMIT=5
 
 # test whether output of the command contains expected lines
 # arguments
 # 1-st command to run
 # 2-nd array of expected strings in the command output
-# 3-rd argument is an optional command runtime limit
-function testOutput {
-IFS="${PREV_IFS}"
-
-    #run the command
-    if [ $# -ge 3 ]; then
-        $1 > ${TMP_FILE} 2>&1 &
-        CMD_PID=$!
-        sleep $3
-        kill $CMD_PID
-    else
-        $1 > ${TMP_FILE} 2>&1
-    fi
-
+# 3-rd argument is a command runtime limit
+function testExpectedMessage {
```

Comment:
Please fix the description of input arguments for testExpectedMessage and testOutput.